src/main/java/edu/rpi/legup/puzzle/minesweeper/elements/Unset.java

```diff
@@ -0,0 +1,9 @@
+package edu.rpi.legup.puzzle.minesweeper.elements;
+
+import edu.rpi.legup.model.elements.NonPlaceableElement;
+
+public class Unset extends NonPlaceableElement {
+    public Unset() {
+        super("MINE-UNPL-0002", "Unset", "A blank tile", "edu/rpi/legup/images/fillapix/tiles/UnknownTile.png");
```

Comment:
I would prefer if you guys made a copy of the Fillapix unknown tile and put it in the Minesweeper directory. All the other puzzles have their own unknown tile images.